layout/session.liquid

```diff
@@ -7,6 +7,7 @@
     <meta name="theme-color" content="{%- if settings.color_schemes.first.settings.background_accent != blank -%}{{- settings.color_schemes.first.settings.background_accent -}}{%- else -%}{{- branding_color -}}{%- endif -%}">
     {% comment %} Base stylesheets {% endcomment %}
     {{ "base.css" | asset_url | stylesheet_tag }}
+    {{ "rx.css" | asset_url | stylesheet_tag }}
```

Comment:
It might be wise to think about moving this CSS file to be shipped with `content_for_header` not to require it to be embedded by the external developers when they will be involved